com.github.tenderowl.norka.json

```diff
@@ -49,28 +50,28 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/libhandy.git",
-                    "tag" : "1.5.0"
+                    "tag": "1.8.2"
                 }
             ]
         },
         {
-            "name" : "gtksourceview",
-            "buildsystem" : "meson",
-            "sources" : [
+            "name": "gtksourceview",
+            "buildsystem": "meson",
+            "sources": [
                 {
-                    "type" : "git",
-                    "url" : "https://gitlab.gnome.org/GNOME/gtksourceview.git",
-                    "tag" : "4.8.2"
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gtksourceview.git",
+                    "tag": "4.8.4"
```

Comment:
gtk3 branch seems to depend on gtksourceview-5 https://github.com/TenderOwl/Norka/blob/402ea912089ed61cef248ee349c51c4b0a9ed54c/meson.build#LL23C2-L23C2 ?

Reply:
Nah, nevermind, I messed up with branches. I decided to merge gtk3 into the master and then rebuild flatpak.